config/default.json.dist

```diff
@@ -6,5 +6,10 @@
       "baseUrl": "your URL",
       "route": "'your route'",
       "listener": "your listener"
+      "filterby": "your filter",
+      "nameofloadtimegauge": "name of your loadtime gauge",
```

Comment:
I would prefer `loadTimeGaugeName` in 🐫  case 
Same for the other ones

Reply:
👌